blocks/graph.py

```diff
@@ -161,8 +161,37 @@ def replace(self, replacements):
             substitutes.
 
         """
-        return ComputationGraph(theano.clone(self.outputs,
-                                             replace=replacements))
+        # Due to bug in theano we have to make one replacement in time
```

Comment:
Indeed, I had problems with `theano.clone` and it is nice that you are also aware of that. Is there a Theano issue for the bug? It would be very nice to put a reference here.